internal/overlord/servstate/manager.go

```diff
@@ -477,3 +477,74 @@ func (m *ServiceManager) CheckFailed(name string) {
 		}
 	}
 }
+
+var shutdownTimeout = killWait + 100*time.Millisecond
+
+// Shutdown shuts down the service manager, stopping all running services.
+func (m *ServiceManager) Shutdown() error {
+	services, err := m.servicesToStop()
+	if err != nil {
+		return err
+	}
+	if len(services) == 0 {
+		logger.Debugf("No services to stop.")
+		return nil
+	}
+
+	// One change to stop them all.
+	logger.Noticef("Stopping all running services.")
+	st := m.state
+	st.Lock()
+	taskSet, err := Stop(st, services)
+	if err != nil {
+		st.Unlock()
+		return err
+	}
+	chg := st.NewChange("shutdown", "Shut down service manager")
```

Comment:
The service manager is an implementation detail which is supposed to be invisible to the user. To the user, what's going on here is that all services are going down, so maybe "Stop all services"?

Reply:
Changed to "Stop all running services".

Also changed the change kind to just "stop" -- it's part of the daemon shutdown sequence, but with the refactoring this change is really just stopping services like an ordinary "stop" change.